### DISCUSSION TITLE
Blanks in File

### DISCUSSION THREAD

**Discussion by cassievann** [CMSgov/hospital-price-transparency#102] [General]
Do we need to fill in any of the blanks in our MRF? Previously, we were audited and told to list N/A. If we are to leave it blank, should we add a generic comment that explains why (ex: something to the effect of "N/A listed as there is not a payer negotiated rate for this charge.")?

**Comment by knh762002** [comment]
This text is listed in the CSV documentation...

"Do not insert a value or any type of indicators (e.g., “N/A”) if the hospital does not have applicable data to encode. If you would like to include an explanation for the blanks, you may do so using Additional Generic Notes or Additional Payer-Specific Notes."

https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV

Hopefully this helps move you in the right direction before somebody from CMS is able to respond.
